src/transformers/integrations.py

```diff
@@ -932,57 +932,50 @@ def __del__(self):
 
 
 class NeptuneCallback(TrainerCallback):
-    """
-    TODO: NPT-12189 - Update the docstring
-    A [`TrainerCallback`] that sends the logs to [Neptune](https://neptune.ai).
+    """TrainerCallback that sends the logs to [Neptune](https://neptune.ai).
 
     Args:
-        api_token ('str', *optional*):
-            Neptune API token obtained from <https://neptune.ai> upon registration.
-            It is recommended to keep it in the `NEPTUNE_API_TOKEN` environment variable
-            -- in this case it does not have to be provided here.
-            Read more in the
-            `Neptune installation docs <https://docs.neptune.ai/getting-started/installation>`_.
-        project (`str`, *optional*):
-            Name of the project. Format: "WORKSPACE/PROJECT".
-            If `None` (default), the value of `NEPTUNE_PROJECT` environment variable will be used.
-            You need to create the project in <https://neptune.ai> first.
-        name (`str`, *optional*):
-            Name of the run, it appears in the "all metadata/sys" section in Neptune UI.
-        base_namespace (`str`, *optional*, defaults to "finetuning"):
-            Root namespace within Neptune's run.
-        log_parameters (`bool`, *optional*, defaults to True):
-            If True, log all trainer arguments and model parameters provided by the trainer.
-        log_checkpoints (`str`, *optional*, defaults to "best"):
-            If "same", upload checkpoints whenever they are saved by the trainer.
-            If "best", upload the best checkpoint (among the ones saved by the trainer,
-            chosen at the end of the training)
-            Otherwise, do not upload checkpoints.
-        run (`Run`, *optional*):
-            Pass Neptune run object if you want to continue logging to the existing run (e.g., resume a run).
-            Read more about it `here <https://docs.neptune.ai/how-to-guides/neptune-api/resume-run>`_.
-            When run object is passed you can't specify other neptune properties.
-        **neptune_run_kwargs (*optional*):
+        api_token (`str`, optional):
+            Neptune API token obtained upon registration. You can leave this argument out if you have
+            saved your token to the `NEPTUNE_API_TOKEN` environment variable (strongly recommended).
+            See full setup instructions in the [docs](https://docs.neptune.ai/getting-started/installation).
+        project (`str`, optional):
+            Name of an existing Neptune project, in the form: "workspace-name/project-name".
+            You can find and copy the name from the project Settings -> Properties in Neptune.
+            If None (default), the value of the `NEPTUNE_PROJECT` environment variable will be used.
+        name (`str`, optional):
+            Custom name for the run. It is shown in the Neptune app and stored in the "sys/name" field.
```

Comment:
Seems that "and stored in the "sys/name" field." is a bit too detailed. At this point users do not know about fields and `sys` namespaces.